### DISCUSSION TITLE
minio update error

### DISCUSSION THREAD

**Discussion by kendn1993** [minio/minio#17512] [Q&A] [answered]
## NOTE
If this case is urgent, please subscribe to [Subnet](https://min.io/pricing) so that our 24/7 support team may help you faster.

After manually changing the jar package and upgrading to the mini version, I encountered a problem of not being able to start,any idea how to deal with this situation?
## Current Behavior

WARNING: Detected Linux kernel version older than 4.0.0 release, there are some known potential performance problems with this kernel version. MinIO recommends a minimum of 4.x.x linux kernel version for best performance
ERROR Unable to initialize backend: /home/minio/data drive is already being used in another erasure deployment. (Number of drives specified: 1 but the number of drives found in the 1st drive's format.json: 4)

![image](https://github.com/minio/minio/assets/27141040/3fe4d3c8-cac3-4a8d-a64f-26517fa8501f)

## Steps to Reproduce (for bugs)

1. Manually update the mini version to the latest version

2. then start all the server

## Your Environment

* Version used (`minio --version`):
the before version mc version RELEASE.2020-08-08T02-33-58Z
Version in use minio version RELEASE.2023-06-09T07-32-12Z (commit-id=6b7c98bd0fde630c6aee78d702b9d37b0200f52c)

* Server setup and configuration:
![image](https://github.com/minio/minio/assets/27141040/5df39cdd-fcf2-40d5-a355-45ff5bebc30c)

* Operating System and version (`uname -a`):
Linux hadoop001 3.10.0-1127.18.2.el7.x86_64 #1 SMP Sun Jul 26 15:27:06 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

**Comment by harshavardhana** [comment, marked as answer]
@kendn1993 it looks like you have not specified all the drives to MinIO.

You have only specified a single drive. Please specify all the drives in the deployment for the server to start.

**Comment by stefanofornari** [comment]
I am running into the same problem. The setup is exactly like when I started minio for the first time, so why should I add drives or so? I am just trying to restart the server. How can I overcome the problem?

**Comment by KierenB** [comment]
I know this is an old thread, but I also have the warning being reported in SNSD mode. Bare metal is a Synology DS916+, running via Docker.
